src/mono/mono/mini/mini-arm64.c

```diff
@@ -620,62 +625,131 @@ emit_subx_sp_imm (guint8 *code, int imm)
 	return code;
 }
 
+
+static int 
+num_trailing_zeros (guint64 imm)
```

Comment:
Can we put this in a header in mono/utils (or in minipal) so we can use it in more places